### PR TITLE
make mongo event string to enums

### PR DESCRIPTION
**Pull request by snys98**
make mongo event string to enums

### CODE REVIEW

**Comment by snys98** [comment]
sorry, but I just made some minimal changes, maybe it's nothing to do with my commit?

**Comment by shaunburdick** [comment]
> sorry, but I just made some minimal changes, maybe the failing check has nothing to do with my commit?

Looks like there are some prettier issues with the change you made to Types.ts

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongtype",
-  "version": "3.6.0",
+  "version": "3.7.0",
```
Comment (shaunburdick) [inline package-lock.json]: You can remove this change, we'll bump the version number on release

**Comment by shaunburdick** [comment]
Closing as stale

**Pull request closed without merging**